### PR TITLE
Feature/lga 2897 dockerize project

### PR DESCRIPTION
**Pull request by psweeting1**
## What does this pull request do?

Project no longer works on M1 machines locally. Project needs to dockerized in order to allow developer to work on the project. 

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist
[LGA-2897](https://dsdmoj.atlassian.net/browse/LGA-2897)

[LGA-2897]: https://dsdmoj.atlassian.net/browse/LGA-2897?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Pheonnexx — APPROVED** [review]
all checked and running on my machine

**Review by BenMillar-CGI — APPROVED** [review]
Looks good to me! 👍 

Note: We have discussed removing the Google Maps API Key, however, this key is exposed to the frontend. It can only be ran on whitelisted domains.

**Review by digitalali-moj — DISMISSED** [review]
Works a charm, thanks!

**Review by TawneeOwl — DISMISSED** [review]
LGTM and works on my machine

File: docs/development.md [hunk 1]
```diff
@@ -25,3 +25,19 @@ To run them manually:
 ```
 pre-commit run --all-files
 ```
+
+# Running Docker locally
+
+## Before beginning
+You will need to have `cla_backend` Docker containers up and running.
+
+`cla_backend` has it's on Docker containers which can be run locally.
```
Comment (davidread) [inline docs/development.md:34]:
```suggestion
`cla_backend` has it's own Docker containers which can be run locally.
```

**Review by Pheonnexx — APPROVED** [review]

**Review by TawneeOwl — APPROVED** [review]
LGTM!

**Pull request merged**